app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -181,7 +183,8 @@ class ExplorationActivityTest {
   ) {
     launch(ExplorationInjectionActivity::class.java).use {
       it.onActivity { activity ->
-        networkConnectionUtil = activity.networkConnectionUtil
+        debugNetworkConnectionUtil =
+          activity.networkConnectionUtil as DebugNetworkConnectionUtil
```

Comment:
Is this actually needed? It seems redundant.

Reply:
Removed.